### DISCUSSION TITLE
customize grains per edge

### DISCUSSION THREAD

**Discussion by MROHOMAN** [neperfepx/neper#936] [Q&A]
Hello,

Is it possible to customize the amount of grains on the edges? I have a 20x20x20 mesh and I want to have exactly 3 equiaxed grains per edge. By using the attached commands I can generate a 20x20x20 mesh with equiaxed grains but I do not have control over the amount of grains per edge.

![image](https://github.com/user-attachments/assets/bfa9a18f-7a0a-4d3e-a398-aa67253bae28)
![image](https://github.com/user-attachments/assets/93bd8acf-bd6f-416b-ae8f-539a7f8f8565)

**Comment by rquey** [comment]
No, it is not.
